src/tenable/load_assets.py

```diff
@@ -22,8 +22,8 @@
 ON DUPLICATE KEY UPDATE uuid=uuid;"""
 
 
-def load_tenable_assets(export: AssetExportStatus, to: mariadb.Connection):
-    conn = to
+def load_tenable_assets(export: AssetExportStatus):
+    conn = mariadb.connect(**CONN_PARAMS)
```

Comment:
Instead of removing the `Connection` parameter, pass in a `ConnectionPool` which this function can obtain a connection from.